PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -312,9 +725,30 @@ struct HfTaskFlow {
     return true;
   }
 
+  template <CorrelationContainer::CFStep step, typename TTrack>
+  bool isMcParticleSelected(TTrack& track)
+  {
+    //  remove MC particles with charge = 0
+    int8_t sign = 0;
+    TParticlePDG* pdgparticle = pdg->GetParticle(track.pdgCode());
+    if (pdgparticle != nullptr) {
+      sign = (pdgparticle->Charge() > 0) ? 1.0 : ((pdgparticle->Charge() < 0) ? -1.0 : 0.0);
+    }
+    if (sign == 0) {
+      return false;
+    }
```

Comment:
Why don't you just check directly `pdgparticle->Charge() == 0`?

Reply:
No specific reason, so I use directly the charge equal 0 now

Reply:
After further investigation, using pdgparticle->Charge() == 0 directly leads to a segmentation fault, so I came back to the previous version

Reply:
That is very unlikely.